src/xcode/ENA/ENA/Source/HTTPClient/Service/RestServiceProvider.swift

```diff
@@ -69,6 +69,19 @@ class RestServiceProvider: RestServiceProviding {
 		}
 	}
 
+	func cached<R>(
+		_ resource: R,
+		_ completion: @escaping (Result<R.Receive.ReceiveModel, ServiceError<R.CustomError>>) -> Void
+	) where R: Resource {
```

Comment:
I'm wondering why this has an asynchronous API and if it needs to have that

Reply:
This is because we use it for the async load function to pass in the completion closure. I'll have a look if we can change that a bit.